misc/roundtrip

```diff
@@ -43,9 +43,9 @@ fi
 limit_entries()
 {
     if [ -n "${MAX_ENTRIES}" ]; then
-	head -${MAX_ENTRIES}
+	    head -${MAX_ENTRIES}
     else
-	cat
+	    cat
```

Comment:
Cold you remove this change from the commit?

If you find/think the indentation style of this line really wrong, you can make a separate commit and add to this pull request.